docs/metric_deletion_retention.md

```diff
@@ -0,0 +1,47 @@
+# Metric Deletion 
+
+Promscale exposes an API for deletion of series. This works same as prometheus delete API.
+
+### Promscale Delete API
+
+URL query parameters:
+
+* **match[]=<series_selector>**: Repeated label matcher argument that selects the series to delete. At least one match[] argument must be provided.
+* **start=<rfc3339 | unix_timestamp>**: Start timestamp. Optional and defaults to minimum possible time.
+* **end=<rfc3339 | unix_timestamp>**: End timestamp. Optional and defaults to maximum possible time.
+
+```
+POST /delete_series
+PUT /delete_series
+```
+
+Example:
+
+```
+curl -X POST -g http://promscale:9201/delete_series?match[]=container_cpu_usage_seconds_total
+```
+
+# Metric Retention
```

Comment:
We should add a todo to support metric (not series) deletion

Reply:
Currently, deletion API supports metric & series deletion, right? 
IMO 
metric: `go_goroutines`
series: `go_goroutines{instance: "localhost:9090"}`

https://stackoverflow.com/questions/49859360/how-do-i-delete-a-time-series-from-prometheus-v2-specifically-a-series-of-alert

Reply:
> We should add a todo to support metric (not series) deletion

@cevian Why is that required? Can't just users do `match[]={__name__="<metric_name>"}` or simply `match[]=<metric_name>`?